openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py

```diff
@@ -2869,9 +2869,10 @@ def ppe_bar1(base):
     #                           (cond2 & (base > ppe.seuil2) & (base <= ppe.seuil3)) * ((ppe.seuil3 - base) * ppe.taux2) +
     #                           (cond2 & (base > ppe.seuil4) & (base <= ppe.seuil5)) * (ppe.seuil5 - base) * ppe.taux3)
             return (1 / ppe_coef) * (
-                ((base <= ppe.seuil2)) * (base) * ppe.taux1
-                + ((base > ppe.seuil2) & (base <= ppe.seuil3)) * (ppe.seuil3 - base) * ppe.taux2
-                + ligne2 * ((base > ppe.seuil4) & (base <= ppe.seuil5)) * (ppe.seuil5 - base) * ppe.taux3)
+                (base <= ppe.seuil2) * (base) * ppe.taux1 +
+                (base > ppe.seuil2) * (base <= ppe.seuil3) * (ppe.seuil3 - base) * ppe.taux2 +
+                ligne2 * (base > ppe.seuil4) * (base <= ppe.seuil5) * (ppe.seuil5 - base) * ppe.taux3
+                )
```

Comment:
Au passage, enquête utilisateur : 

la fonction [`select`](https://doc.openfisca.fr/en/coding-the-legislation/30_case_disjunction.html#multiples-conditions) offre une alternative à cette gestion très mathématique des disjonctions de cas.

Est-ce que tu ne l'as pas utilisé parce que tu ne la connais pas, ou parce que la syntaxe te paraît moins lisible. Une critique que l'on fait souvent à `select` est qu'elle sépare les paires conditions / valeurs. Est-ce que c'est un gros inconvénient pour toi ?